docs/tutorials/vmc-from-scratch.ipynb

```diff
@@ -0,0 +1,2173 @@
+{
```

Comment:
<p>formulas not rendered properly </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/1586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='99'/>